data/releasenotes.json

```
@@ -1,4 +1,17 @@
 [
+	{
+		"version": "alpha/2024-11-21",
+		"date": "2024-11-21",
+		"breaksdb": false,
+		"notes": [
+			"<b>New Feature:</b> Add support for <tt>psql</tt> builtin describe commands <tt>\\d table</tt> and <tt>\\d+ table</tt>.",
+			"<b>PostgreSQL compatibility:</b> Include attribute defaults in the <tt>pg_attrdef</tt> system table.",
+			"<b>Maintenance:</b> Fixed a rare deadlock during concurrent compression of inserts.",
```

```suggestion
			"<b>Maintenance:</b> Fix a rare deadlock during concurrent compression of inserts.",
```
Wenn wir überall sonst auch present tense verwenden.